### PR TITLE
enh(Java): more optimized way of bubble-sort

### PR DESCRIPTION
**Pull request by shivam-0510**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [x] Optimization in previous algorithms
- [x] Code style update
- [x] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

### CODE REVIEW

**Comment by ming-tsai** [comment]
> **What kind of change does this PR introduce?** (check at least one)
> 
> * [x]  Bugfix
> * [x]  New algorithm
> * [x]  Optimization in previous algorithms
> * [x]  Code style update
> * [x]  Refactor
> * [x]  Documentation
> * [x]  Other, please describe:
> 
> **Briefly describe the changes in this PR**
> 
> **Other information:**

Hi @shivam-0510,
Do not select all checklist, in your case, this is not a new algorithm and if you checked other please describe something

**Comment by shivam-0510** [comment]
Hi @ming-tsai ,
I had unmarked the unnecessary checklist.

File: algorithms/Java/sorting/bubble-sort.java [hunk 1]
```diff
@@ -1,27 +1,41 @@
+package JAVA;
```
Comment (TawfikYasser) [inline algorithms/Java/sorting/bubble-sort.java:1]: Welcome @shivam-0510 

Thanks for the good work

Please remove this line `package JAVA;` as it is only required for your local project.

and please add some information about the new optimized algorithm in a multi-line comment.

**Review by TawfikYasser — CHANGES_REQUESTED** [review]

**Review by TawfikYasser — APPROVED** [review]
Nice work @shivam-0510

**Comment by shivam-0510** [comment]
Thanks for the appreciation @TawfikYasser

**Pull request merged**